src/components/pages/home-page/home-page.tsx

```diff
@@ -1,89 +1,290 @@
-                            Manage your organization&apos;s roster with the built-in suite of tools designed to allow for
-                            quick and easy modifications.
-                          </Typography>
-                        </li>
-                      )}
-                      {user.activeRole?.role.canViewMuster && (
-                        <li>
-                          <Typography>
-                            <strong><Link to="/muster">Muster Snapshot</Link></strong><br />
-                            Track your group&apos;s daily muster compliance, view weekly and monthly trends, as well as
-                            export data.
-                          </Typography>
-                        </li>
-                      )}
-                      {user.activeRole?.role.workspaces && (
-                        <li>
-                          <Typography>
-                            <strong><Link href={`/dashboard?orgId=${orgId}`}>Customizable Dashboards</Link></strong><br />
-                            Protect sensitive data by creating and assigning role-based permissions across the entire
-                            organization.
-                          </Typography>
-                        </li>
-                      )}
-                      <li>
-                        <Typography>
-                          <strong><Link to="/settings">Alerts &amp; Notifications</Link></strong><br />
-                          Stay up-to-date on organization activity without overburdening your inbox or mobile device
-                          with fully customizable alerts.
-                        </Typography>
-                      </li>
-                    </ul>
-                  </Grid>
-
-                  <Grid item xs={4}>
-                    <img src={welcomeImage} height="300" alt="Welcome" />
-                  </Grid>
-                </Grid>
+                <Typography className={classes.metricLabel}>
+                  User Access Requests
+                </Typography>
+                <Typography className={classes.metricValue}>
+                  {accessRequests.length}
+                </Typography>
+                <Link to="/users">View Now &rarr;</Link>
               </CardContent>
             </Card>
           </Grid>
+          {user.activeRole?.role.workspaces && (
+            <Grid item xs={12}>
+              <Box display="flex" alignItems="center" justifyContent="space-between">
+                <Box display="flex" alignItems="center" className={classes.favoritesContainer}>
+                  <FavoriteIcon color="secondary" />
+                  <Typography variant="subtitle1">
+                    My Favorited Dashboards
+                  </Typography>
+                </Box>
+                <Link to="/workspaces">View All Dashboards</Link>
+              </Box>
+              {workspaces.map(workspace => (
```

Comment:
If there are no favorite dashboards, perhaps some kind of message should be shown.